std_msgs/Header header
int32 idx

sensor_msgs/CompressedImage infra1_image

sensor_msgs/CompressedImage infra2_image

sensor_msgs/Image depth_image

sensor_msgs/CompressedImage rgb_image

sensor_msgs/Image ouster_depth_image

sensor_msgs/PointCloud2 ouster_pcl2
